### PR TITLE
Support for memory in decimal GiB

### PR DESCRIPTION
**Pull request by qiao-summer**
colima start --cpu 1 --memory 0.8 --disk 10

### CODE REVIEW

File: environment/vm/lima/yaml.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ func newConf(ctx context.Context, conf config.Config) (l limaconfig.Config, err
 		l.CPUs = &conf.CPU
 	}
 	if conf.Memory > 0 {
-		l.Memory = fmt.Sprintf("%dGiB", conf.Memory)
+		l.Memory = fmt.Sprintf("%dMiB", uint32(conf.Memory*1024)) //memorySize is a multiple of 1 megabyte
```
Comment (abiosoft) [inline environment/vm/lima/yaml.go:63]: The comment is not needed.

```suggestion
		l.Memory = fmt.Sprintf("%dMiB", uint32(conf.Memory*1024))
```

File: environment/vm/lima/limautil/instance.go [hunk 2]
```diff
@@ -18,13 +18,13 @@ func Instance() (InstanceInfo, error) {
...
-	Arch    string `json:"arch,omitempty"`
-	CPU     int    `json:"cpus,omitempty"`
-	Memory  int64  `json:"memory,omitempty"`
-	Disk    int64  `json:"disk,omitempty"`
-	Dir     string `json:"dir,omitempty"`
+	Name    string  `json:"name,omitempty"`
+	Status  string  `json:"status,omitempty"`
+	Arch    string  `json:"arch,omitempty"`
+	CPU     int     `json:"cpus,omitempty"`
+	Memory  float32 `json:"memory,omitempty"`
```
Comment (abiosoft) [inline environment/vm/lima/limautil/instance.go:25]: This should not be changed to float, it is the representation of the memory in bytes.

File: app/app.go [hunk 3]
```diff
@@ -293,17 +293,17 @@ func (c colimaApp) SSH(args ...string) error {
...
+	DisplayName  string  `json:"display_name"`
+	Driver       string  `json:"driver"`
+	Arch         string  `json:"arch"`
+	Runtime      string  `json:"runtime"`
+	MountType    string  `json:"mount_type"`
+	IPAddress    string  `json:"ip_address"`
+	DockerSocket string  `json:"docker_socket"`
+	Kubernetes   bool    `json:"kubernetes"`
+	CPU          int     `json:"cpu"`
+	Memory       float32 `json:"memory"`
```
Comment (abiosoft) [inline app/app.go:305]: This should not be changed to float, it is representation of memory in bytes.

**Comment by qiao-summer** [comment]
@abiosoft you said “This should not be changed to float, it is representation of memory in bytes.”，i don not konw how to modify this field type

**Comment by qiao-summer** [comment]
@abiosoft I resubmitted with another pull request

**Pull request closed without merging**